### PR TITLE
Also accept $FRAMEWORK_HOST as a valid URI in airgap_linter (#2256)

### PR DESCRIPTION
**Pull request by benclarkwood**

### CODE REVIEW

File: tools/airgap_linter.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ def is_bad_uri(uri, file_name):
         ".mesos:",
         "$MESOS_CONTAINER_IP",
         "{{FRAMEWORK_HOST}}",
+        "$FRAMEWORK_HOST",
```
Comment (elezar) [inline tools/airgap_linter.py]: Looking at how these are matched to the URIs, that means that the following:
* `$MESOS_CONTAINER_IP_BUT_NOT_REALLY`
* `$FRAMEWORK_HOST_BUT_ALSO_NOT_QUITE`
will also be accepted as valid URIs.

Also, in terms of environment variables: `${FRAMEWORK_HOST}` and `${MESOS_CONTAINER_IP}` are also valid usages of environment variables.

Comment (benclarkwood) [reply tools/airgap_linter.py]: That is a good point. I will take a pass at writing a better URL evaluator.

I will add both of those alternate versions.

Comment (benclarkwood) [reply tools/airgap_linter.py]: (in a follow up)

Comment (elezar) [reply tools/airgap_linter.py]: Ok. That's fine.

**Pull request merged**